scripts/jenkins/cloud/ansible/roles/ssh_keys/tasks/main.yml

```diff
@@ -25,3 +25,15 @@
       {{ key }}
       {% endfor %}
       {% endfor %}
+
+- name: Create local SSH key on deployer
```

Comment:
I'm not sure we want to generate keys on the developers machine without asking.

Reply:
I would recommend at least adding a note on the documentation about it. Also the key is created only if the default key does not exist right?

Reply:
Ok, good point. I could add a variable that enables the key generation so that a user has to set it to get a new key.